tests/logprob/test_basic.py

```diff
@@ -205,9 +211,10 @@ def test_joint_logprob_subtensor():
     I_value_var = I_rv.type()
     I_value_var.name = "I_value"
 
-    A_idx_logp = joint_logprob({A_idx: A_idx_value_var, I_rv: I_value_var}, sum=False)
+    A_idx_logp = factorized_joint_logprob({A_idx: A_idx_value_var, I_rv: I_value_var})
```

Comment:
This test should be moved to `logprob/mixture.py` or removed if there's already an equivalent one

Reply:
Some tests in `test_mixture.py` use indexing but none explicitly check for a valid indexing operation. So I think it is good to simply move it there.